app/views/layouts/email.html.erb

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
     <title>C2</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width" />
     <link rel="stylesheet" type="text/css" href="<%= stylesheet_url("mailer_new.css") %>" />
+    <%= render 'mail_shared/outlook_fix' %>
```

Comment:
super tiny thing but we're trying to use all double quotes on this proj so `'` -> `"` please!